.github/workflows/test.yml

```
@@ -24,7 +24,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04, windows-latest, macos-latest]
+        os: [ubuntu-18.04]
+        # os: [ubuntu-18.04, windows-latest, macos-latest]
```

```suggestion
        os: [ubuntu-latest, windows-latest, macos-latest]
```

@joerick Not sure what you want to do for our own testing - latest or pinned.

---

sure, let's commit this

---

I think latest for our own CI, we want to catch incompatibilities I think